lang/pl/home.json

```
@@ -81,10 +81,10 @@
       "Kontekst",
       "Suspense",
       "Granice błędów",
```

```suggestion
      "Error Boundaries",
```
Zmieniamy, czy leży to w granicach błędu? 🤣

---

Zostawiłem "Granice błędów", ponieważ z tego co patrzyłem zarówno w dokumentacji Reacta jak i w innych źródłach zwykle takie sformułowanie jest używane zamiast oryginalnego. Jak dziwnie by nie brzmiało to chyba jest to sama grupa docelowa odbiorców :smile: